src/main/python/community-detector/fixtures/requirements.txt

```diff
@@ -0,0 +1,5 @@
+python-igraph==0.7.1.post6
+scipy==1.0.1
+cassandra_driver >= 3.12.0, <4.0
```

Comment:
Do we use Cassandra somewhere in fixture generation?

Reply:
because apollo is not available in pip, it is cloned in ./apollo. Instead of installing apollo/requirements.txt, the minimum dependencies to call apollo are included here.

Without cassandra driver, it breaks with:
```
Traceback (most recent call last):
  File "generator.py", line 5, in <module>
    from apollo.graph import ConnectedComponentsModel, CommunitiesModel, detect_communities
  File "./apollo/apollo/graph.py", line 17, in <module>
    from apollo.cassandra_utils import get_db, patch_tables, BatchedHashResolver, Session
  File "./apollo/apollo/cassandra_utils.py", line 9, in <module>
    from cassandra.cluster import Cluster, Session, NoHostAvailable
ModuleNotFoundError: No module named 'cassandra'
```

Reply:
This is only for tests generation, so it's fine.

But this makes me think that may be all of `ConnectedComponentsModel`, `CommunitiesModel ` and even `CommunityDetector` are a candidates to be proposed refactored out of `graph.py` (to minimize dependency footprint) in the future.